src/sample/Models/QuestionContainer.java

```diff
@@ -0,0 +1,8 @@
+package sample.Models;
+
+import java.util.HashMap;
+
+public class QuestionContainer {
+    private HashMap<Answer, Integer> containerOfAllQuestions;
```

Comment:
What's the purpose of the `Integer` here? Also wasn't the idea to store all questions in a collection, not all answers?